javaparser-core/src/main/java/com/github/javaparser/ast/body/ModifierSet.java

```diff
@@ -19,8 +19,11 @@
 package com.github.javaparser.ast.body;
 
 import com.github.javaparser.ast.AccessSpecifier;
+import com.github.javaparser.ast.Modifiers;
+import com.github.javaparser.ast.NodeWithModifiers;
 
 import java.lang.reflect.Modifier;
```

Comment:
the problem is that for compatibility I have to specify and accept also integers, but I cannot find the value for the Default Modifier (it is not present in j.l.r.Modifier and looking on the JVM Specs I did not find it so far)